gamebook/output.py

```diff
@@ -0,0 +1,3 @@
+class Output(object):
+    def __init__(self):
+        raise NotImplementedError
```

Comment:
You also need a notimplemented for the `output` function, otherwise you didn't really do anything.